honeybee_radiance/modifier/material/mist.py

```diff
@@ -28,15 +28,15 @@ class Mist(Material):
     medium (no absorption). The scattering eccentricity parameter will likewise override
     the global setting if it is present. Scattering eccentricity indicates how much
     scattered light favors the forward direction, as fit by the Henyey-Greenstein
-    function:
+    function::
```

Comment:
Does the formula show up ok with just the `::`? If so, this would also be something good to document in the wiki.

Also I realized that this class also lacks `Args` and `Properties`.

Reply:
Yes, this is a variation of the `.. code-block:: shell` directive. I can document these options together.